scripts/ci-configmap.sh

```diff
@@ -25,5 +25,5 @@ make --directory="${REPO_ROOT}" "${KUBECTL##*/}"
 CM_NAMES=("calico-addon" "calico-ipv6-addon" "calico-dual-stack-addon" "calico-windows-addon")
 CM_FILES=("calico.yaml" "calico-ipv6.yaml" "calico-dual-stack.yaml" "windows/calico")
 for i in "${!CM_NAMES[@]}"; do
-	"${KUBECTL}" create configmap "${CM_NAMES[i]}" --from-file="${REPO_ROOT}/templates/addons/${CM_FILES[i]}" --dry-run -o yaml | kubectl apply -f -
+	"${KUBECTL}" create configmap "${CM_NAMES[i]}" --from-file="${REPO_ROOT}/templates/addons/${CM_FILES[i]}" --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
I'll make sure to re-apply this after #2495 I believe I removed that line entirely

Reply:
done https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2495/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R294